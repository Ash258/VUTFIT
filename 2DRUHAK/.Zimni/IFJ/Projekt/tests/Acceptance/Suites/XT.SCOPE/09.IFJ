' Sequence of scopes

SCOPE
	dim a as integer = 5

	SCOPE
		dim a as integer = 10
		print a;
	END SCOPE

	SCOPE
		dim a as integer = 20
		print a;
	END SCOPE


	SCOPE
		dim a as integer = 30
		print a;
	END SCOPE

	print a;
END SCOPE
