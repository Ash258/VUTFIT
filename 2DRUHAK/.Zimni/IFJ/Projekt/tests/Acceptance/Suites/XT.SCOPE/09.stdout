 10 20 30 5